core/src/main/scala/org/broadinstitute/dsde/workbench/hamm/db/WorkflowTableQueries.scala

```diff
@@ -0,0 +1,130 @@
+package org.broadinstitute.dsde.workbench.hamm
+package db
+
+import java.sql.ResultSet
+import java.time.Instant
+
+import scalikejdbc._
+import io.circe._
+import io.circe.parser._
+import io.circe.syntax._
+import org.broadinstitute.dsde.workbench.hamm.model._
+import org.postgresql.util.PGobject
+
+
```

Comment:
Scalike queries for the Workflow table:

- insert workflow
- get workflow
- get workflow cost

The LabelTypeBinder and and LabelPbf are binders for jsonb